packaging/rpm/microshift.spec

```diff
@@ -162,8 +163,26 @@ cp ./_output/bin/${GOOS}_${GOARCH}/microshift-etcd ./_output/microshift-etcd
 
 # SELinux modules build
 
-cd packaging/selinux
+pushd packaging/selinux
 make
+popd
+
+# osbuilder sample blueprints build
+function create_blueprint() {
+  local -r larch="$1"
+
+  REPLACE_USHIFT_VERSION="%{version}" \
+    REPLACE_USHIFT_ARCH="${larch}" \
+    envsubst < "packaging/blueprint/blueprint.toml.template" > "packaging/blueprint/blueprint-${larch}.toml"
+
+  jq -r \
```

Comment:
I added the jq and gettext (envsubst) build requirement to `release-info`.

I'm not sure we should introduce the "knowledge" of blueprint format, etc. to rebase.sh. I'd rather have it processed when building RPMs.

Reply:
@dhellmann , I had a discussion on this with @pmtk and he pointed out that we can only substitute image references in rebase.sh, but not the RPM version. For this reason, I think we may want to keep the current implementation.
WDYT?

Reply:
OK, that's a good point.

Why do we need gettext as a build dependency?

Reply:
Because of envsubst usage

```
$ sudo dnf whatprovides envsubst
gettext-0.21-7.el9.x86_64 : GNU libraries and utilities for producing multi-lingual messages
Repo        : @System
Matched from:
Filename    : /usr/bin/envsubst
```

Reply:
Ha, I had to fix the rpm-podman builds because of the right dependencies in the spec.
It's working great!